index.js

```diff
@@ -1,12 +1,14 @@
 'use strict'
 
+require('rootpath')()
```

Comment:
I understand that this breaks convention but I should also mention that rootpath isn't there to only remove the leading ```./```. It actually allows you (read: you don't have to use it; you can also just specify the path like you are used to)  to specify paths relative to the root of the project. For me it makes a lot more sense to write ```require('index.js')``` instead of (potentially) ```require('../../../index.js')```  when requiring ```index.js``` in the project root (IMHO the latter path specification is ugly as hell and also much easier to get wrong if your editor doesn't help you with resolving this path). This is why I would like to see this module being used in strider.
However, I am not overly attached to this module (or absolute paths for that matter; i just find it easier to read). 
Do you still think I should get rid of this module?